### ISSUE TITLE
Inquiry about the version of pytorch lightning

### ISSUE THREAD

**Issue by hyc9** [FoundationVision/OmniTokenizer#7]
Thank you for your excellent work!  I've tried switching between a few versions of pytorch lightning, but I'm encountering conflicts here or there.   I wonder what version is being used in the code?

**Comment by hyc9** [comment]
I tried the version in requirements.txt( i.e. 1.5.4) too and there are still some conflicts

**Comment by hyc9** [comment]
Here are some error lines encountered in different versions, for example :
`parser = pl.Trainer.add_argparse_args(parser) #with pl 2.x.x
from pytorch_lightning.utilities.rank_zero import rank_zero_only #with pl 1.5.4
#with 1.8.1 / 1.9.5
return [opt_ae, opt_disc], [{"scheduler": sch_ae, "interval": "step"}, {"scheduler": sch_disc, "interval": "step"}] 
#or [{"optimizer": opt_ae, "scheduler": sch_ae, "interval": "step"}, {"optimizer": opt_disc, "scheduler": sch_disc, "interval": "step"}]

sch1, sch2 = self.lr_schedulers() 
`

**Comment by wdrink** [comment]
<img width="805" alt="截屏2024-06-26 09 20 22" src="https://github.com/FoundationVision/OmniTokenizer/assets/48371342/3fc92208-9696-4165-b92b-5cc9ebefff96">

I use pl == 1.5.4, it seems to work well for me.
